wiki/Contests/Official_support/en.md

```diff
@@ -62,7 +62,11 @@ If your contest satisfies the [eligibility criteria](#eligibility), you can make
   - Previous iterations of the same contest series (if applicable).
   - Your contest on the [Mappers' Guild contest listing](https://mappersguild.com/contests/listing) (if applicable).
 
-Once we receive your request, the osu! team will send an approval or denial of your contest's official support status. Please send any such requests as early as possible before registrations, ideally at least a month in advance, to allow time for consideration.
+Once we receive your request, the osu! team will send an approval or denial of your contest's official support status.
+
+Profile badge prizes must be requested at most two months after the contest's completion. **Badges will never be approved until after the conclusion of the contest.**
+
+For rews posts, setting up contest listings, or any other inquiries, please send such requests as early as possible before registrations, ideally at least a month in advance, to allow time for consideration.
```

Comment:
```suggestion
For news posts, setting up contest listings, or any other inquiries, please send such requests as early as possible before registrations, ideally at least a month in advance, to allow time for consideration.
```

Reply:
i FIXED the rews thing multiple times idk why it isn't saving i'm gonna LOSE it